src/wwwroot/js/genpage/gentab/presets.js

```diff
@@ -292,16 +293,91 @@ function editPreset(preset) {
     fixPresetParamClickables();
 }
 
+let presetSortByElement;
+let presetSortReverseElement;
+let presetSortBy = localStorage.getItem('preset_list_sort_by') || 'Default';
+let presetSortReverse = localStorage.getItem('preset_list_sort_reverse') === 'true';
+
+
+function onPresetSortByChanged(ev) {
+    localStorage.setItem('preset_list_sort_by', presetSortBy = ev.target.value);
+    presetBrowser.update();
+}
+
+function onPresetSortReverseChanged(ev) {
+    localStorage.setItem('preset_list_sort_reverse', presetSortReverse = ev.target.checked);
+    presetBrowser.update();
+}
+
+function getSortInfo() {
+    const sortByElem = document.getElementById("preset_list_sort_by");
+    const reverseElem = document.getElementById("preset_list_sort_reverse");
+
+    if (sortByElem !== presetSortByElement) {
+        presetSortByElement?.removeEventListener("change", onPresetSortByChanged);
+        presetSortByElement = sortByElem;
+        if (sortByElem) {
+            sortByElem.value = presetSortBy;
+            sortByElem.addEventListener("change", onPresetSortByChanged);
+        }
+    }
+
+    if (reverseElem !== presetSortReverseElement) {
+        presetSortReverseElement?.removeEventListener("change", onPresetSortReverseChanged);
+        presetSortReverseElement = reverseElem;
+        if (reverseElem) {
+            reverseElem.checked = presetSortReverse;
+            reverseElem.addEventListener("change", onPresetSortReverseChanged);
+        }
+    }
+    
+    return {sortBy: presetSortBy, reverse: presetSortReverse};
+}
+
+function getSortCompareFunc(sortBy, reverse) {
+    const getValue = preset => {
+        if (preset.title.toLowerCase() === "default") {
+            return "          default";
+        }
+        if (preset.title.toLowerCase() === "preview") {
+            return "          preview";
+        }
+        
+        switch (sortBy) {
+            // for Name, use title, but strip off the path part
+            case 'Name': return preset.title.substring(preset.title.lastIndexOf('/') + 1);
+            case 'Title': return preset.title;
```

Comment:
'name' / 'title' split here is odd

Reply:
I should rename "Title" -> "Path" to hopefully be more clear that one sorts by just the basename and one by the full path (which is called Title in the UI).